include/vm_map.h

```diff
@@ -22,6 +27,9 @@ typedef struct vm_map {
   SPLAY_HEAD(vm_map_tree, vm_map_entry) tree;
   size_t nentries;
   pmap_t *pmap;
+  /* sbrk support */
+  vm_map_entry_t *td_brk_entry; /* The entry where brk segment resides in. */
```

Comment:
`brk` and `sbrk` change data segment size, so I suppose those two fields should be named `data_entry`, `data_size`.